torch/export/exported_program.py

```diff
@@ -413,7 +423,14 @@ def _decompose_and_get_gm_with_new_signature_constants(
                                     entry
                                 ]
 
-        with fake_mode:
+        with fake_mode, _override_composite_implicit_decomp(
+            # Preserve variants of aten::to understanding that they are mutating/aliasing
+            # and their CompositeImplicitAutograd kernels will not become NotImplemented.
+            # We will later replace them with aten._to_copy when functionalizing.
+            (torch.ops.aten.to.dtype_layout, torch.ops.aten.to.dtype),
```

Comment:
Could you actually make it a seperate context manager? In my PR, I am merging this path so that it always calls export_to_aten_ir and it already takes many flags as function call.

Reply:
I'll create an alias.